### DISCUSSION TITLE
Next Js Apakah termasuk framework fullstack?

### DISCUSSION THREAD

**Discussion by Mikael2909** [deaafrizal/forum#55] [The Lounge]
Mau nanya nih, kemarin saya ada mengikuti playlist bang Dea buat website anime pakai next Js, yang mau saya tanya, sebenarnya next Js itu termasuk framework fullstack ya? Soalnya saya lihat di konten bang Dea ada menggunakan mysql sebagai backend nya
Mohon pencerahannya teman-teman semua

**Comment by andikaalakate** [comment]
Iyaa, bang. Termasuk Framework Fullstack

**Reply by Mikael2909** [reply]
oke, makasih ya

**Reply by andikaalakate** [reply]
Siipp
